packages/eslint-plugin/src/rules/prefer-function-type.ts

```diff
@@ -18,6 +18,8 @@ export default util.createRule({
     messages: {
       functionTypeOverCallableType:
         "{{ type }} has only a call signature - use '{{ sigSuggestion }}' instead.",
```

Comment:
Slightly unrelated to your change, but whilst we're here....

I think it's probably better to not include the signature in the error message for this one. Lets simplify this - we have an auto fixer so no need to add noise to the error.

Also, might be better to change the parameter to be `typeName` so it aligns with your new message? (I think that's much better than just `type`)

```suggestion
        "'{{ typeName }}' only has a call signature, you should use a function type instead.",
```

Reply:
here `type` is referring to the string `'Type literal'` or  `'Interface'` where my message refers to the interface name, so I think I'll rename my message to use `interfaceName` and rename this to be `literalOrInterface` to be clearer. And will remove suggestion from message.